ReactSkia/components/RSkComponentImage.cpp

```diff
@@ -373,6 +373,8 @@ void RSkComponentImage::requestNetworkImageData(ImageSource source) {
   auto sharedCurlNetworking = CurlNetworking::sharedCurlNetworking();
   networkRequestMutex_.lock();
   if((isRequestInProgress_ == true) && (remoteCurlRequest_!=nullptr)){
+    // if url is changed, image component is get component property update.
```

Comment:
TYpo errors

Reply:
we may not need extra variable to store oldUri. The curl request already stores the URL we can compare against it .